gaia-button.js

```diff
@@ -24,6 +24,24 @@ module.exports = component.register('gaia-button', {
     this.setupShadowRoot();
     this.setAttribute('role', 'button');
     this.tabIndex = 0;
+    if (this.keyboardEnabled) {
+      this.addEventListener('keydown', this._handleKeyEvent.bind(this));
+      this.addEventListener('keyup', this._handleKeyEvent.bind(this));
+    }
+  },
+
+  _handleKeyEvent: function(e) {
+    if (e.keyCode !== KeyEvent.DOM_VK_RETURN) {
+      return;
+    }
+
+    if (e.type === 'keydown') {
+      this.toggled = true;
+    } else if (e.type === 'keyup') {
+      this.toggled = false;
+      this.setAttribute('released', '');
```

Comment:
release attribute is for "animation". There are four states of a button: normal, focused, pressed released. In TV, we have different animation style for the transitions of some states:
* normal => focused (we call this focusing),
* focused => pressed (we call this pressing),
* pressed => released (we call this releasing),
* released => focused (we call this released to focusing)

If `focusing` and `released to focusing` have the same transition style, we don't need  released state. But they are different.

At gaia button, we don't have focused state. So, we should bring this state to gaia-button. We only need it at gaia-tv-button.